### DISCUSSION TITLE
I only get one frame when I try to play /dev/video0 with ffplay

### DISCUSSION THREAD

**Discussion by CluelessTechnologist** [umlaeute/v4l2loopback#539] [General]
It doesn't seem to matter what source I use, ffplay shows a few frames and then freezes on one frame.
This is the command I use to play the virtual webcam feed:
`ffplay -pixel_format yuyv422 -i /dev/video0`

So far I have tried these sources:

Capturing my DSLR via gphoto, seems to work when piping it directly to ffplay, altough a bit laggy:
`gphoto2 --stdout --capture-movie | ffmpeg -i - -vcodec rawvideo -pix_fmt yuv420p -f v4l2 /dev/video0`

Streaming Big Buck Bunny to /dev/video0, works perfectly when using ffplay directly:
`ffmpeg -i Downloads/Big\ Buck\ Bunny\ -\ FULL\ HD\ 60FPS.mp4  -vf format=yuv420p -f v4l2 /dev/video0`

**Kernel:** `6.1.0-6-amd64`
**modinfo for v4l2loopback:**
```
filename:       /lib/modules/6.1.0-6-amd64/extra/v4l2loopback.ko
alias:          char-major-10-255
license:        GPL
version:        0.12.7-321-gfb410fc
author:         Vasily Levin, IOhannes m zmoelnig <zmoelnig@iem.at>,Stefan Diewald,Anton Novikovet al.
description:    V4L2 loopback video device
srcversion:     8ACC385CF489014F8DA5399
depends:        videodev
retpoline:      Y
name:           v4l2loopback
vermagic:       6.1.0-6-amd64 SMP preempt mod_unload modversions
parm:           debug:debugging level (higher values == more verbose) (int)
parm:           max_buffers:how many buffers should be allocated [DEFAULT: 2] (int)
parm:           max_openers:how many users can open the loopback device [DEFAULT: 10] (int)
parm:           devices:how many devices should be created (int)
parm:           video_nr:video device numbers (-1=auto, 0=/dev/video0, etc.) (array of int)
parm:           card_label:card labels for each device (array of charp)
parm:           exclusive_caps:whether to announce OUTPUT/CAPTURE capabilities exclusively or not  [DEFAULT: 0] (array of bool)
parm:           max_width:maximum allowed frame width [DEFAULT: 8192] (int)
parm:           max_height:maximum allowed frame height [DEFAULT: 8192] (int)
```
Any ideas on what could be wrong?

**Comment by umlaeute** [comment]
try again with current `main` (1d01a4156cd22ff25d015079cbb731078fa89ead)

**Reply by CluelessTechnologist** [reply]
Wow, now it works! What changed between the commit I was running and https://github.com/umlaeute/v4l2loopback/commit/1d01a4156cd22ff25d015079cbb731078fa89ead?
Thanks for fixing the issue!

**Comment by umlaeute** [comment]
What happened is https://github.com/umlaeute/v4l2loopback/pull/536
